src/iosrv/init.c

```diff
@@ -427,7 +432,7 @@ abt_init(int argc, char *argv[])
 	/* Now, initialize Argobots. */
 	rc = ABT_init(argc, argv);
 	if (rc != ABT_SUCCESS) {
-		D_ERROR("failed to init ABT: %d\n", rc);
+		D_ERROR("failed to init ABT: "DF_RC"\n", DP_RC(rc));
```

Comment:
ABT will not return daos err.  See dss_abterr2derr below

Reply:
right.